### PR TITLE
Changed path to Suave DLLs

### PR DESCRIPTION
**Pull request by pblasucci**
Changed path to Suave DLLs

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ let startWebServer () =
         Writers.set_header "Cache-Control" "no-cache, no-store, must-revalidate"
         >>= Writers.set_header "Pragma" "no-cache"
         >>= Writers.set_header "Expires" "0"
-        >>= browse
+        >>= browse'
```
Comment (forki) [inline build.fsx]: Is this correct?

**Comment by pblasucci** [comment]
@forki I was a bit taken back by it as well... seems to be a change in the latest version of Suave.

**Comment by haf** [comment]
Yes, I wrote about it in the release notes: https://github.com/SuaveIO/suave/releases/tag/v0.19.1

You might also be interested in reading the latest: https://github.com/SuaveIO/suave/releases/tag/v0.20.0 as they introduce a change in how forms are parsed; for this we would love community input.

I'm also thinking of releasing 0.21 after dinner or tomorrow with a 'host' applicative.

**Comment by haf** [comment]
Sorry, here https://github.com/SuaveIO/suave/releases/tag/v0.18.0

**Pull request merged**